scripts/schema-publish.sh

```diff
@@ -6,15 +6,69 @@
 
 schemaDir="src/schemas/validation"
 branch=$(git branch --show-current)
-version=${branch:1:3}
-echo === Building schemas into ./deploy/oas/$version
+
+if [[ $branch =~ ^v([0-9]+\.[0-9]+\.[0-9]+) ]]; then
```

Comment:
```suggestion
if [[ $branch =~ ^v([0-9]+\.[0-9]+)-dev$ ]]; then
```
The deploy/oas subfolders are named 3.0, 3.1, etc., only MAJOR.MINOR.

Reply:
I adjusted the capture group to only grab MAJOR MINOR, while still demanding the presence of PATCH.
Hence, dev branches are still expected to be named `vX.Y.Z-whatever`.